boards/arduino/arduino_uno_r4_minima/Kconfig.arduino_uno_r4_minima

```diff
@@ -2,5 +2,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_ARDUINO_UNO_R4_MINIMA
-	bool "Arduino Uno R4 Minima board"
-	depends on SOC_R7FA4M1AB3CFM
+	select SOC_R7FA4M1AB3CFM
```

Comment:
Is this change actually related to Renesas platform?

Reply:
It's a renesas chip, so yes?